### PR TITLE
[Merged by Bors] - Document config overrides

### LINKED ISSUE #330: Document supported config overrides
Most settings can be overridden today already even though they are not exposed as fields in the CRD.
This functionality however is hidden and not documented.

Please document all the possible files, CLI and Env (if applicable) that can be overridden.

You can take a look at the Druid PR: https://github.com/stackabletech/druid-operator/pull/154
But please feel free to improve if you find anything.

### PR DESCRIPTION
**Pull request by teozkr**
## Description

Fixes #330, based on https://github.com/stackabletech/druid-operator/pull/154

## Review Checklist
- [ ] Code contains useful comments
- [ ] (Integration-)Test cases added (or not applicable)
- [ ] Documentation added (or not applicable)
- [ ] Changelog updated (or not applicable)
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
- [ ] Helm chart can be installed and deployed operator works (or not applicable)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: docs/modules/ROOT/pages/usage.adoc [hunk 1]
```diff
@@ -55,3 +55,67 @@ If you wish to include integration with https://docs.stackable.tech/opa/index.ht
...
+      auto.create.topics.enable: "false"
+  roleGroups:
+    default:
+      replicas: 1
+----
+
+All override property values must be strings.
+
+For a full list of configuration options we refer to the Apache ZooKeeper https://zookeeper.apache.org/doc/r3.7.0/zookeeperAdmin.html#sc_configuration[Configuration Reference].
+
```
Comment (maltesander) [inline docs/modules/ROOT/pages/usage.adoc:96]: artifact?

Comment (teozkr) [reply docs/modules/ROOT/pages/usage.adoc:96]: Argh, I thought I had updated that.

Comment (teozkr) [reply docs/modules/ROOT/pages/usage.adoc]: Fixed

**Review by maltesander — APPROVED** [review]
LGTM.

**Pull request closed without merging**